protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/generation/ProtobufBuildOrchestrator.java

```diff
@@ -140,6 +140,7 @@ public boolean generate(GenerationRequest request) throws ResolutionException, I
         .addPlugins(resolvedPlugins, request.getOutputDirectory())
         .addSourcePaths(compilableSources)
         .setFatalWarnings(request.isFatalWarnings())
+        .setDescriptorFile(request.getDescriptorFile())
```

Comment:
Do we want to update the check on line 99 so we don't fail if we just want to produce a protobuf descriptor file?

Reply:
Check added